plaid/api.go

```diff
@@ -0,0 +1,27 @@
+package plaid
+
+type APIResponse struct {
+	RequestID string `json:"request_id"`
+}
+
+type SecretAPIRequestJson struct {
+	ClientID    string `json:"client_id"`
+	Secret      string `json:"secret"`
+	AccessToken string `json:"access_token"`
+}
+
+// Credentials and parameters to be used for testing.
+var (
+	TEST_CLIENT_ID  = "5a1dce3d4e95b836d37e3397"
```

Comment:
This stuff shouldn't be checked in - could you please amend this commit and remove these fields? You should be able to specify these environment variables in the CI suite.

Reply:
updated!